### DISCUSSION TITLE
Checkout succeeding, but fails when LFS flag enabled

### DISCUSSION THREAD

**Discussion by nicutus** [community/community#160533] [Actions]
I've recently gotten access to an instance of the self-hosted enterprise GitHub and have been tasked with getting a runner working for some eventual CI/CD work. GitHub is on version 3.8 and I have a self-hosted runner on an ubuntu 20.04 LTS instance. I've successfully gotten the runner installed and connected. I can clone/checkout repositories using the action/checkout@v3 method. However, I'm unable to use the lfs flag for that command. I end up receiving "LFS: Client error: filepath" and "Error: error: failed to fetch some objects from 'myrepo/info/lfs'" (where filepath is the path to the lfs files)

The repository checks out fine but of course none of the lfs files have been fetched. I've tried a few things to try and figure out what's going on, but with no luck. The only method I've found that actually 'works' is to run the workflow and have it checkout (without the lfs flag) and then navigate to the checked out repo and run a git lfs fetch from the terminal. It prompts me for a username and password (I provide it my username and an access token as we use SSO to sign on) and it fetches the lfs files. From then on, I can enable the lfs flag with no errors.

I have tried providing the same token via the 'token' flag for the checkout action, but I get the same error as if I didn't give it the token. I've also tried a few different ways of using the local git credential manager to store the token and login info just to try to get things to work, but that hasn't worked either. It seems to me like there's an authentication issue, but I'm not entirely sure why or what. I could also be completely off base and am just confused. Any help would be greatly appreciated! :)

**Comment by unlobito** [comment]
Heya 👋 If you find this discussion and still need help with `git-lfs`, check out https://github.com/actions/checkout/issues/415#issuecomment-2143700018 for a workaround.
